app/api/models/genome.py

```diff
@@ -63,23 +63,26 @@ def serialize_url(self, url: str):
         return url
 
 
-class GenomeDetails(BaseModel):
+class BriefGenomeDetails(BaseModel):
```

Comment:
During code-review @azangru mentioned that soon  BriefGenomeDetails will have additional fields that aren't required in GenomeDetails. 
Proposal for this:

`
class BaseGenomeDetails(BaseModel):
    genome_id: str = Field(alias="genomeUuid")
    genome_tag: Optional[str] = Field(
        alias=AliasChoices(
            AliasPath("assembly", "urlName"), AliasPath("assembly", "tolId")
        ),
        default=None,
    )
    common_name: str = Field(alias=AliasPath("organism", "commonName"), default=None)
    scientific_name: str = Field(alias=AliasPath("organism", "scientificName"))
    type: Optional[Type] = None
    is_reference: bool = Field(
        alias=AliasPath("assembly", "isReference"), default=False
    )
    assembly: AssemblyInGenome = None
`

`
class BriefGenomeDetails(BaseGenomeDetails):
   //available_datasets: list[str]
   pass
`
`class GenomeDetails(BaseGenomeDetails):
//Rest of GenomeDetails goes here
`

Reply:
@Mehrnaz-Charkhchi , @azangru Thank you both, that was a good idea, I've done the changes